regexes/device/mobiles.yml

```diff
@@ -10378,9 +10384,15 @@ Vitelcom:
     - regex: 'portalmmm/[12].0 (TSM[a-z0-9 ]+)'
       model: '$1'
 
+# Fengxiang
+Fengxiang:
+  regex: '(vKB011B|vKB004L)'
+  device: 'tablet'
+  model: '$1'
+
 # VK Mobile
 VK Mobile:
-  regex: 'VK(?!share)[\-]?([a-z0-9 ]+)[ ;\)/]'
+  regex: 'VK(?!share)[\-]?([a-zA-Z0-9 ]+)([;\)/ ]| Build)'
```

Comment:
the `A-Z` should actually not be needed, as all regexes are done case insensitive in https://github.com/matomo-org/device-detector/blob/master/Parser/ParserAbstract.php#L180

Reply:
`A-Z` removed